blocks/layout-grid/src/grid/edit.native.js

```diff
@@ -53,7 +53,19 @@ function ColumnsEdit( {
 } ) {
 	const { verticalAlignment, align } = attributes;
 
-	const [ isDefaultColumns, setDefaultColumns ] = useState( ! columns );
+	const [ isDefaultColumns, setDefaultColumns ] = useState( false );
+	if ( ! columns ) {
+		// Set the default column on the next tick.
+		// This is done so that the insertion of the Layout Grid Block inside other blocks
+		// such as thr group block or itself doesn't conflict with removal BlockPicker BottomSheet
+		// and the VariationControl BottomSheet.
+		// And we end up with a state where the VariationControl is set to be visible
+		// but not dismissible.
+		// eslint-disable-next-line @wordpress/react-no-unsafe-timeout
+		setTimeout( () => {
```

Comment:
In [Columns](https://github.com/WordPress/gutenberg/blob/efc28f91fb2c4924b173bc7c8f60dbffac62a7d2/packages/block-library/src/columns/edit.native.js#L119-L124) we utilize [useEffect](https://reactjs.org/docs/hooks-effect.html) to do something similar. I wonder if we can use that for a more precise event trigger.

Reply:
Good point :+1: 

I just tried that and unfortunately, it didn't have the same results. :(

Reply:
Bummer thanks for trying it out.